misc/org.palladiosimulator.codeconventions/palladio-checkstyle-rules-8.29.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<!--
+    This configuration file was written by the eclipse-cs plugin configuration editor
+-->
+<!--
+    Checkstyle-Configuration: Palladio Coding Conventions
+    Description: none
+-->
+<module name="Checker">
+  <property name="severity" value="warning"/>
+  <module name="LineLength">
+    <property name="severity" value="info"/>
+    <property name="ignorePattern" value="\*|import"/>
+    <property name="max" value="130"/>
+  </module>
+  <module name="TreeWalker">
+    <property name="tabWidth" value="4"/>
+    <module name="JavadocMethod">
```

Comment:
I think the properties of this check were removed in 8:28  see [Github issue](https://github.com/checkstyle/checkstyle/issues/7329)
Therefore we also should remove them